src/sentry/api/endpoints/organization_releases.py

```diff
@@ -464,12 +466,15 @@ def post(self, request: Request, organization) -> Response:
             result = serializer.validated_data
             scope.set_tag("version", result["version"])
 
-            allowed_projects = {p.slug: p for p in self.get_projects(request, organization)}
+            projects_from_request = self.get_projects(request, organization)
```

Comment:
I took a look at `get_projects` but I couldn't figure out how this would work when the `projects` is a list of slugs and IDs. Did you take a look at the function?

Reply:
discussed offline